processor/probabilisticsamplerprocessor/README.md

```diff
@@ -35,6 +35,51 @@ processors:
     sampling_percentage: 15.3
 ```
 
+The probabilistic sampler supports sampling logs according to their trace ID, or by a specific log record attribute.
+
+The probabilistic sampler optionally may use a `hash_seed` to compute the hash of a log record.
+This sampler samples based on hash values determined by log records. In order for
+log record hashing to work, all collectors for a given tier (e.g. behind the same load balancer)
+must have the same `hash_seed`. It is also possible to leverage a different `hash_seed` at
+different collector tiers to support additional sampling requirements. Please refer to
+[config.go](./config.go) for the config spec.
+
+The following configuration options can be modified:
+- `hash_seed` (no default, optional): An integer used to compute the hash algorithm. Note that all collectors for a given tier (e.g. behind the same load balancer) should have the same hash_seed.
+- `sampling_percentage` (required): Percentage at which logs are sampled; >= 100 samples all logs, 0 rejects all logs.
+- `trace_id_sampling` (default = true, optional): Whether to use the log record trace ID to sample the log record.
+- `sampling_source` (default = null, optional): The optional name of a log record attribute used for sampling purposes, such as a unique log record ID. The value of the attribute is only used if the trace ID is absent or if `trace_id_sampling` is set to `false`.
+- `sampling_priority` (default = null, optional): The optional name of a log record attribute used to set a different sampling priority from the `sampling_percentage` setting. 0 means to never sample the log record, and >= 100 means to always sample the log record.
+
+Examples:
+
+Sample 15% of the logs:
+```yaml
+processors:
+  probabilistic_sampler:
+    sampling_percentage: 15
+```
+
+Sample logs according to their logID attribute:
+
+```yaml
+processors:
+  probabilistic_sampler:
+    sampling_percentage: 15
+    trace_id_sampling: false
```

Comment:
How about something like:

```
processors:
  probabilistic_sampler:
    sampling_percentage: 15
    attribute_source: record # possible values: one of context, resource, traceID, record
    from_attribute: logID # value is required if the source is not traceID  
```

At first, you could have `record` and `traceID` only, expanding to other sources in the future. This nomenclature is inline with other processors, like the routing processor: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/routingprocessor#configuration

Reply:
This is nicer. Adopting it.